src/integTest/resources/legacy-syntax/android/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+  repositories {
+    maven { url 'https://maven.google.com' }
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:2.3.0'
+  }
+}
+
+plugins {
+  id 'org.sonatype.gradle.plugins.scan'
+}
+
+allprojects {
+  repositories {
+    maven { url 'https://maven.google.com' }
+    mavenCentral()
+  }
+}
+
+ext {
+  compileSdkVersion=28
+  minSdkVersion=10
+  targetSdkVersion=28
+  buildToolsVersion='26.0.2'
+}
+
+nexusIQScan {
+  serverUrl = 'http://localhost:8081'
+  username = 'admin'
+  password = 'admin123'
+  applicationId = 'testing-gradle-plugin'
+  simulationEnabled = true
+}
+
+ossIndexAudit {
+  simulationEnabled = true
```

Comment:
Question. Is it worth having `colorEnabled = false` here?

Reply:
> Question. Is it worth having `colorEnabled = false` here?

Integration tests are made using `contains` instead of exact comparisons because we have to use the full output of the plugin, so having the extra ascii color characters hasn't been an issue.

At the moment I see no reason to add that flag.